woocommerce/woocommerce-admin
# Analytics Flag: Remove performance indicators when disabled

Performance indicators depend on APIs and data stores that are turned off by disabling Analytics. Although the UI works on `main`, it will display outdated data if orders have arrived after the Analytics flag has been disabled.

I decided to remove the entire Stats Overview widget because the only element not associated with Analytics is the Jetpack connection call to action. In the absence of a design, I think its best to remove the whole widget.

### Detailed test instructions:

1. Disable Analytics `wp option set woocommerce_analytics_enabled no`
2. View the homescreen. The Stats Overview section should not be there.
3. If the task list is hidden or complete, a double column layout will be shown with Store Management links on the right.
4. If the task list is shown, the Store Management links will be hidden and a single column layout will be shown.
